tests/test_other_scripts.py

```diff
@@ -113,6 +114,34 @@ def test_vacuum(self):
                 assert valid2[key] == valid[key], f"{key} score doesn't match"
                 assert test2[key] == test[key], f"{key} score doesn't match"
 
+    def test_vacuum_nobackup(self):
+        with testing_utils.tempdir() as tmpdir:
+            from parlai.scripts.vacuum import Vacuum
+
+            model_file = os.path.join(tmpdir, 'model')
+            valid, test = testing_utils.train_model(
+                {
+                    'task': 'integration_tests',
+                    'optimizer': 'adam',
+                    'learningrate': 0.01,
+                    'model_file': model_file,
+                    'num_epochs': 0.01,
+                    'no_cuda': True,
+                    'skip_generation': True,
+                    'batchsize': 8,
+                    # TODO: switch to test_agents/unigram
+                    'model': 'transformer/generator',
+                    'ffn_size': 8,
+                    'embedding_size': 8,
+                    'n_layers': 1,
+                }
+            )
+            size_before = os.stat(model_file).st_size
+            Vacuum.main(model_file=model_file, no_backup=True)
+            size_after = os.stat(model_file).st_size
+            assert size_after < size_before
```

Comment:
nit: I usually prefer having that string after `assert` that says what went wrong. Although you can find it later since it is inside a test, and test report tells you what failed. But it might save you a couple of minutes to see it in the stderr. 
A diagnosis tip might help too.

Reply:
I agree but context is narrow and clear